### PR TITLE
Support RequestBody

### PR DESCRIPTION
**Pull request by leolplex**
Hello @StefH,

I'm trying to add the support to request body, I'm following the same logic used in the response, I'm facing a problem when I try to use `BodyAsJson` in the `RequestModel` 'cause it does not exist, I'm using `Body` but it does not work as I expected, could you have idea why it is happening? 

I appreciate your help.

### CODE REVIEW

**Comment by StefH** [comment]
Can you merge/rebase master into this branch ?

**Comment by leolplex** [comment]
Hello,

Please skip my first comment I found a solution cast the object with `JsonConvert.SerializeObject` and change the matcher to `JsonMatcher`, please check the last commit.
 Thanks.

**Comment by StefH** [comment]
@leolplex 
Can you take a look at my comments?

**Comment by leolplex** [comment]
> @leolplex Can you take a look at my comments?

@StefH
 I am no able to see your comments.

**Review by StefH — CHANGES_REQUESTED** [review]

**Comment by leolplex** [comment]
@StefH comments applied!

File: src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs [hunk 1]
```diff
@@ -82,6 +101,20 @@ private MappingModel MapOperationToMappingModel(string path, string httpMethod,
...
         }
 
+        private BodyModel MapRequestBody(object requestBody)
+        {
+            if (requestBody == null)
+            {
+                return null;
+            }
+
+            BodyModel bodyRequestModel = new BodyModel();
```
Comment (StefH) [inline src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs:111]: `var requestBodyModel = new BodyModel();`

**Review by StefH — CHANGES_REQUESTED** [review]

File: src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs (hunk 1 above)
Comment (leolplex) [reply src/WireMock.Net.OpenApiParser/Mappers/OpenApiPathsMapper.cs]: @StefH Done!

**Review by StefH — APPROVED** [review]

**Comment by StefH** [comment]
@leolplex 
I'll merge tomorrow and create a new NuGet

**Comment by leolplex** [comment]
> @leolplex I'll merge tomorrow and create a new NuGet

Thanks a lot @StefH  👍

**Pull request merged**